docs/COMPILE-NIX.md

```diff
@@ -61,9 +61,14 @@ To build Unicorn on *nix (such as MacOSX, Linux, *BSD, Solaris):
 
         $ sudo ./make.sh install
 
-  For FreeBSD/OpenBSD, where sudo is unavailable, run:
+  For OpenBSD, where sudo is unavailable, run:
 
         $ su; ./make.sh install
+        
+  On FreeBSD, system's `make` is different from GNU make, so you need to install it.
+  
+        # pkg install gmake
+        $ gmake
```

Comment:
i think `make.sh` still works for FreeBSD, due to these lines https://github.com/unicorn-engine/unicorn/blob/master/make.sh#L89-L92.

can you confirm that you can compile like this on FreeBSD?

```
$ ./make.sh
$ sudo ./make.sh install
```